### PR TITLE
[13.0][IMP] ddmrp add product vendor code

### PR DESCRIPTION
**Pull request by TDu**
Add the product vendor code on the stock buffer based on the already
computed main supplier.

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

**Review by mmequignon — APPROVED** [review]

File: ddmrp/models/stock_buffer.py [hunk 1]
```diff
@@ -804,6 +804,18 @@ def _compute_main_supplier(self):
             else:
                 rec.main_supplier_id = False
 
+    @api.depends("main_supplier_id", "product_id.seller_ids")
+    def _compute_product_vendor_code(self):
+        for rec in self:
+            if not rec.main_supplier_id:
+                rec.product_vendor_code = False
+                continue
+            supplier_info = rec.product_id.seller_ids.filtered(
```
Comment (LoisRForgeFlow) [inline ddmrp/models/stock_buffer.py]: Could you adjust to use the method `_get_product_sellers`?

Comment (TDu) [reply ddmrp/models/stock_buffer.py]: Done

**Comment by LoisRForgeFlow** [comment]
@TDu LGTM, can you squash your commits so I can merge?

**Comment by TDu** [comment]
@LoisRForgeFlow Thanks and squashed

**Comment by LoisRForgeFlow** [comment]
Ty!

/ocabot merge minor

**Comment by LoisRForgeFlow** [comment]
@TDu Could you forward port this to v14? thanks!

**Pull request merged**